extensions/sql-database-projects/src/sqldbproj.d.ts

```diff
@@ -152,6 +153,12 @@ declare module 'sqldbproj' {
 		 */
 		addPostDeploymentScript(relativePath: string): Promise<void>;
 
+		/**
```

Comment:
FYI @sergten, if you want to be able to add non-build items (e.g. files that aren't sql scripts) to projects).